### PR TITLE
Update envoy response flags that we know about

### PR DESCRIPTION
**Pull request by jshaughn**
Update envoy response flags that we know about for the Flags tab in graph side-panel.

Closes https://github.com/kiali/kiali/issues/3685

### CODE REVIEW

**Comment by pbajjuri20** [comment]
Tested and passed on Kiali UI 1.30.0 (b5fef87824ec670d43871d9a9874489f2cd7df94)

**Review by pbajjuri20 — APPROVED** [review]

File: src/utils/ResponseFlags.ts [hunk 1]
```diff
@@ -1,15 +1,20 @@
-// The Envoy flags can be found here:
-// https://github.com/envoyproxy/envoy/blob/master/source/common/stream_info/utility.cc
+// The Envoy response flags can be found here:
+// https://www.envoyproxy.io/docs/envoy/latest/configuration/observability/access_log/usage
```
Comment (jmazzitelli) [inline src/utils/ResponseFlags.ts:2]: I don't see how this link is relevant, compared to the old one. The old utility.cc I can see the 2-char codes.. but I don't see those in the new link.

Comment (jmazzitelli) [reply src/utils/ResponseFlags.ts:2]: Oh, I see.. %RESPONSE_FLAGS%... should probably mention that in the comments since it is half-way down on the page.

**Review by jmazzitelli — APPROVED** [review]
I think the comment should be alittle more descriptive, but that's trivial ... approving..

**Review by jmazzitelli — APPROVED** [review]

**Pull request merged**